packages/block-editor/src/components/image-size-control/index.js

```diff
@@ -50,33 +52,28 @@ export default function ImageSizeControl( {
 				/>
 			) }
 			{ isResizable && (
-				<div className="block-editor-image-size-control">
-					<p className="block-editor-image-size-control__row">
-						{ __( 'Image dimensions' ) }
-					</p>
-					<div className="block-editor-image-size-control__row">
-						<TextControl
-							type="number"
-							className="block-editor-image-size-control__width"
+				<VStack className="block-editor-image-size-control" spacing="0">
+					<p>{ __( 'Image dimensions' ) }</p>
+
+					<HStack align="baseline" spacing="3">
```

Comment:
What is the align baseline for?

Reply:
To keep the height input aligned. Otherwise, it looks like this: 

<img width="285" alt="Screenshot 2023-01-26 at 9 54 01 PM" src="https://user-images.githubusercontent.com/35543432/215019302-294ee506-0f95-43e7-a271-ab2b0fce6a36.png">

Reply:
So it isn't needed now that we dealt with the bottom margin problem?

Reply:
@mirka, can you clarify what you mean by 'it' in your question? I'm not sure I understand so I just wanted to confirm if I need to remove anything before merging. 🙂

Reply:
Ah sorry, I meant the `align="baseline"`.

Reply:
No worries! It is needed, from what I can tell, so it doesn't end up looking like the screenshot above.